go/contacts/tasks.py

```diff
@@ -114,10 +123,13 @@ def contacts_to_csv(contacts, include_extra=True):
 
 
 def get_group_contacts(contact_store, *groups):
+    # TODO: FIXME: Kill this thing. It keeps all contact objects in memory.
```

Comment:
I'm sure I'm overlooking something, but couldn't we just turn this function into a generator to avoid keeping all contact objects in memory?